doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -319,11 +319,11 @@ The updateContext request payload contains a list of contextElement
 elements. Each contextElement is associated to an entity (whose
 identification is provided in the entityId element, in this case we
```

Comment:
...identification is provided in the entityId element -> ...identification is provided by the `id`, `type` and `isPattern` fields.

Please, review all the file for other ocurrences of "entityId" where the same fix make have sense.

(This is not really part of the changed we discussed to include in this PR, but I have spotted while doing the diff, so I think it make sense to fix it also)